spec/models/dex_connector_ldap_spec.rb

```diff
@@ -18,4 +18,317 @@
         .to include("BEGIN CERTIFICATE")
     end
   end
+
+  describe "#name_validations" do
+    let(:dex_connector_ldap) { create(:dex_connector_ldap) }
+
+    it { is_expected.to validate_presence_of(:name) }
+
+    it "does not allow the name to be empty" do
```

Comment:
You can remove this since you are already using `validate_presence_of` matcher above.